apps/src/templates/projects/projectsModule.js

```diff
@@ -0,0 +1,30 @@
+/** @file Redux actions and reducer for the Projects Gallery */
```

Comment:
just noting that the filename here follows the style used in gamelab, whereas we use `redux.js` in most other parts in apps. I think this is fine and should be easier to switch later if we decide to standardize.